text/0000-cargo-binary-dependencies.md

```diff
@@ -0,0 +1,140 @@
+- Feature Name: (`bindeps`)
+- Start Date: 2020-11-30
+- RFC PR: [rust-lang/rfcs#3028](https://github.com/rust-lang/rfcs/pull/3028)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Allow Cargo packages to depend on `bin`, `cdylib`, and `staticlib` crates, and use the artifacts built by those crates.
```

Comment:
Addition: Current packages depend on `lib` (static or shared Rust libraries) explained [here](https://doc.rust-lang.org/reference/linkage.html).

Reply:
I'm going to put this in the reference section instead of the summary to keep the intro short.